tf/modules/load-balancing/target/main.tf

```diff
@@ -48,8 +48,8 @@ resource "aws_alb_target_group" "service" {
 
   stickiness {
     enabled            = var.stickiness_enabled
-    type               = "app_cookie"
-    cookie_name        = var.stickiness_cookie_name
+    type               = "lb_cookie"
```

Comment:
I don't think we want to hardcode this, it's then restricting this modules use.

Would it be better to allow caller to choose what they want? We want to use `"lb_cookie"` for this particular use case but might want other uses of this to use `"app_cookie"`.